### ISSUE TITLE
Duplicate YAML keys

### ISSUE THREAD

**Issue by jhkennedy** [ASFHyP3/its-live-monitoring#71]
Our last deployment from `develop` failed due to duplicate YAML keys:
```
 while constructing a mapping
  in "<unicode string>", line 62, column 24:
                Condition: !Or
                           ^ (line: 62)
found duplicate key "ArnEquals" with value "OrderedDict([('aws:SourceArn', {'Ref': 'Sentinel2TopicArn'})])" (original value: "OrderedDict([('aws:SourceArn', {'Ref': 'LandsatTopicArn'})])")
  in "<unicode string>", line 65, column 15:
                  ArnEquals:
                  ^ (line: 65)

To suppress this check see:
    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys
```
https://github.com/ASFHyP3/its-live-monitoring/actions/runs/8928243403/job/24523366399

which is due to these lines:
https://github.com/ASFHyP3/its-live-monitoring/blob/develop/cloudformation.yml#L62-L66

Likely the condition syntax just needs to be updated

**Comment by jhkennedy** [comment]
The template change in #70 resulted in a different error:
```
An error occurred (ValidationError) when calling the CreateChangeSet operation: Template error: Fn::Equals cannot be partially collapsed
```
https://github.com/ASFHyP3/its-live-monitoring/actions/runs/8955093260/job/24595443565

I suspect we actually want to use an `ArnLike` IAM condition operator here:
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html#Conditions_ARN

**Comment by jhkennedy** [comment]
Fixed in #73 :tada:
